### DISCUSSION TITLE
question about  integrated TiledVAE

### DISCUSSION THREAD

**Discussion by lizd0413** [lllyasviel/stable-diffusion-webui-forge#384] [Q&A] [answered]
I saw the updates that ControlNet and TiledVAE are integrated, then i ran the update.bat, but i can't find TiledVAE.
What should I do? Should I reinstall forge?

**Comment by lizd0413** [comment]
Another question: Are there any plans to integrate Noise Inversion?

**Reply by ostap667inbox** [reply, marked as answer]
https://github.com/lllyasviel/stable-diffusion-webui-forge/issues/191

**Reply by lizd0413** [reply]
ty <3
